app/components/article-featured-video.js

```diff
@@ -85,5 +119,58 @@ export default Component.extend({
 			path: '/',
 			domain: config.cookieDomain
 		});
-	}
+	},
+
+	didInsertElement() {
+		this.onScrollHandler = this.onScrollHandler.bind(this);
+
+		if (inGroup('FEATURED_VIDEO_VIEWABILITY_VARIANTS', 'ON_SCROLL')) {
+			$(window).on('scroll', this.onScrollHandler);
+		}
+	},
+
+	/**
+	 * FIXME FEATURED VIDEO A/B TEST ONLY
+	 */
+	onScrollHandler() {
+		const currentScrollPosition = window.pageYOffset,
+			requiredScrollDelimiter = this.getScrollDelimiter(),
+			hasScrollClass = this.element.classList.contains(scrollClassName);
+
+		if (currentScrollPosition >= requiredScrollDelimiter && !hasScrollClass) {
+			this.element.classList.add(scrollClassName);
+			this.track(trackActions.impression, 'onscroll');
+		} else if (currentScrollPosition < requiredScrollDelimiter && hasScrollClass) {
+			this.element.classList.remove(scrollClassName);
+		}
+	},
+
+	/**
+	 * Returns the lowest scroll delimiter where video should start floating
+	 *
+	 * @returns {number}
+	 */
+	getScrollDelimiter() {
+		if (this.get('smartBannerVisible') && !this.get('isFandomAppSmartBannerVisible')) {
+			// navbar + smart banner + page header
+			return 220;
```

Comment:
Are you sure all these elements, especially page header, have fixed height? I would use jquery `position` to get these values.